pyphare/pyphare/pharesee/hierarchy.py

```diff
@@ -1502,3 +1522,41 @@ def get_times_from_h5(filepath):
     f.close()
     return times
 
+
+def hierarchy_compare(this, that):
+    if not isinstance(this, PatchHierarchy) or not isinstance(that, PatchHierarchy):
+        return False
```

Comment:
a report class here too with all the `return False`(s) as you can't really tell what failed at runtime with this